File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/create_adjust_packages_request.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class CreateAdjustPackagesRequest(TypedDict, total=False):
    AdjustmentDate: str
    AdjustmentReason: str
    Label: str
    Quantity: int
    ReasonNote: str
    UnitOfMeasure: str

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/CreateAdjustPackagesRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class CreateAdjustPackagesRequest(
    @JsonProperty("AdjustmentDate")
    val adjustmentDate: String? = null,
    @JsonProperty("AdjustmentReason")
    val adjustmentReason: String? = null,
    @JsonProperty("Label")
    val label: String? = null,
    @JsonProperty("Quantity")
    val quantity: Int? = null,
    @JsonProperty("ReasonNote")
    val reasonNote: String? = null,
    @JsonProperty("UnitOfMeasure")
    val unitOfMeasure: String? = null
)
